int8 id

float32 position_x
float32 position_y
float32 position_z

float32 center_x
float32 center_y

float32 width

float32 angle

string color
string shape